File: 백준/Bronze/2338. 긴자리 계산/긴자리 계산.cs
using System;
using System.Numerics;

class Q2338
{
    static void Main(string[] args)
    {
        BigInteger A = BigInteger.Parse(Console.ReadLine());
        BigInteger B = BigInteger.Parse(Console.ReadLine());
        BigInteger sum = A + B;
        BigInteger sub = A - B;
        BigInteger mul = A * B;
        
        Console.WriteLine(sum);
        Console.WriteLine(sub);
        Console.WriteLine(mul);
    }
}

File: 백준/Bronze/2338. 긴자리 계산/긴자리 계산.java
import java.util.Scanner;
import java.math.BigInteger;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        BigInteger A = sc.nextBigInteger();
        BigInteger B = sc.nextBigInteger();
        BigInteger sum = A.add(B);
        BigInteger sub = A.subtract(B);
        BigInteger mul = A.multiply(B);
        
        System.out.println(sum);
        System.out.println(sub);
        System.out.println(mul);
    }
}